python/README.md

```diff
@@ -1,15 +1,23 @@
 # Python Client - Examples and Code Snippets
 
-This folder contains examples and code snippets for the openEO Python client.
+This folder contains code snippets for the openEO Python client.
 
 ## Overview
 
 Environments: `Python` (plain Python code), `Jupyter` (e.g. Notebooks)
 
+**Furthermore in usecase_notebooks repository:**
+
+
 | Title | Environment | Description |
 | ----- | ----------- | ----------- |
-| ...   | `Python`    | ...         |
+| BurntMapping   | `Jupyter`    | Classical Normalized Burnt Ratio(NBR) difference performed using VITO backend on a chunk polygon. The method followed in this notebook to compute DNBR is inspired from [UN SPIDER's recommended practices](https://www.un-spider.org/advisory-support/recommended-practices/recommended-practice-google-earth-engine-flood-mapping). |
```

Comment:
Please add links to the individual folders:

```suggestion
| [BurntMapping](./BurntMapping/)   | `Jupyter`    | Classical Normalized Burnt Ratio(NBR) difference performed using VITO backend on a chunk polygon. The method followed in this notebook to compute DNBR is inspired from [UN SPIDER's recommended practices](https://www.un-spider.org/advisory-support/recommended-practices/recommended-practice-google-earth-engine-flood-mapping). |
```

Reply:
update